x-pack/plugins/alerting/server/alerts_client/lib/build_new_alert.ts

```diff
@@ -57,44 +76,31 @@ export const buildNewAlert = <
     [
       cleanedPayload,
       {
-        '@timestamp': timestamp,
-        event: {
-          action: 'open',
-          kind: 'signal',
-        },
-        kibana: {
-          alert: {
-            action_group: legacyAlert.getScheduledActionOptions()?.actionGroup,
-            flapping: legacyAlert.getFlapping(),
-            flapping_history: legacyAlert.getFlappingHistory(),
-            instance: {
-              id: legacyAlert.getId(),
-            },
-            maintenance_window_ids: legacyAlert.getMaintenanceWindowIds(),
-            rule: rule.kibana?.alert.rule,
-            status: 'active',
-            uuid: legacyAlert.getUuid(),
-            workflow_status: get(cleanedPayload, ALERT_WORKFLOW_STATUS, 'open'),
-            ...(legacyAlert.getState().duration
-              ? { duration: { us: legacyAlert.getState().duration } }
-              : {}),
-            ...(legacyAlert.getState().start
-              ? {
-                  start: legacyAlert.getState().start,
-                  time_range: {
-                    gte: legacyAlert.getState().start,
-                  },
-                }
-              : {}),
-          },
-          space_ids: rule.kibana?.space_ids,
-          version: kibanaVersion,
-        },
-        tags: Array.from(
-          new Set([
-            ...((cleanedPayload?.tags as string[]) ?? []),
-            ...(rule.kibana?.alert.rule.tags ?? []),
-          ])
+        ...rule,
```

Comment:
**question:** Is copying the rule new? as in we should of done that all along?

Reply:
I think we have been doing that